AUTHORS

```diff
@@ -3,3 +3,4 @@ Kartikei Mittal<kartikeimittal@gmail.com>
 Umesh<23umesh.here@gmail.com>
 Rohan Singh<singh.77@iitj.ac.in>
 Tarun Singh Tomar <tomartarun2001@gmail.com>
+Saptashrungi Birajdar <saptashrungib@gmail.com>
```

Comment:
No spaces between name and email. Please update line 5 and 6 as well in your new PR.